src/main/java/com/product/model/Product.java

```diff
@@ -1,17 +1,22 @@
 package com.product.model;
 
+import java.util.Date;
```

Comment:
unnecessary imports

please enable setting on your IDE to get rid of them automatically (or when you format the code)